### ISSUE TITLE
Please add support for FidelityFX Super Resolution 2

### ISSUE THREAD

**Issue by MOVZX** [xCollateral/VulkanMod#136]
Since FSR2 supports Vulkan maybe you guys can add FSR2 into this Minecraft's Mod.

**Comment by LiberteIV** [comment]
Lol, i came here to ask exactly the same question just to see that someone else asked 37 minutes before me.

A plague tale requiem supports DLSS, on nexusmods i found that someone ported Fidelity Super Resolution 2.1.2, basically replacing DLSS with FSR, the result is that with Quality Mode on 2560x1440p i went from 40 FPS to 80 FPS with a 1080 Ti.
This is amazing technology boosting performance like nothing else, and the image looks even better than native solution.

Please add support for FidelityFX Super Resolution 2.1.2

**Comment by xCollateral** [comment]
It will be possible later in the development.

**Comment by MOVZX** [comment]
> Lol, i came here to ask exactly the same question just to see that someone else asked 37 minutes before me.
> 
> A plague tale requiem supports DLSS, on nexusmods i found that someone ported Fidelity Super Resolution 2.1.2, basically replacing DLSS with FSR, the result is that with Quality Mode on 2560x1440p i went from 40 FPS to 80 FPS with a 1080 Ti. This is amazing technology boosting performance like nothing else, and the image looks even better than native solution.
> 
> Please add support for FidelityFX Super Resolution 2.1.2

Do you mean this one: https://www.nexusmods.com/aplaguetalerequiem/mods/9

Because I'm the one who made that Mod page for APTR. And I really wish Minecraft with this Vulkan Mod can implement FSR too.

**Comment by LiberteIV** [comment]
> > Lol, i came here to ask exactly the same question just to see that someone else asked 37 minutes before me.
> > A plague tale requiem supports DLSS, on nexusmods i found that someone ported Fidelity Super Resolution 2.1.2, basically replacing DLSS with FSR, the result is that with Quality Mode on 2560x1440p i went from 40 FPS to 80 FPS with a 1080 Ti. This is amazing technology boosting performance like nothing else, and the image looks even better than native solution.
> > Please add support for FidelityFX Super Resolution 2.1.2
> 
> Do you mean this one: https://www.nexusmods.com/aplaguetalerequiem/mods/9
> 
> Because I'm the one who made that Mod page for APTR. And I really wish Minecraft with this Vulkan Mod can implement FSR too.

Yes, i had a feeling it was you, just wasn't sure, nice that you posted here!
